bootstrap/glob.go

```diff
@@ -131,8 +131,8 @@ func (s *globSingleton) GenerateBuildActions(ctx blueprint.SingletonContext) {
 			depFile := fileListFile + ".d"
 
 			fileList := strings.Join(g.Files, "\n") + "\n"
-			pathtools.WriteFileIfChanged(fileListFile, []byte(fileList), 0666)
-			deptools.WriteDepFile(depFile, fileListFile, g.Deps)
+			pathtools.WriteFileIfChanged(absolutePath(fileListFile), []byte(fileList), 0666)
```

Comment:
Why wasn't this triggering an error that we'd notice?

Reply:
Because it doesn't check the returned error.  I'll fix that too.